src/main/resources/beam-template.conf

```diff
@@ -93,6 +93,15 @@ beam.agentsim.agents.modalBehaviors.mulitnomialLogit.params.walk_intercept = "do
 beam.agentsim.agents.modalBehaviors.mulitnomialLogit.params.bike_intercept = "double | 0.0"
 beam.agentsim.agents.modalBehaviors.mulitnomialLogit.utility_scale_factor = "double | 1.0"
 beam.agentsim.agents.modalBehaviors.lccm.filePath = ${beam.inputDirectory}"/lccm-long.csv"
+beam.agentsim.agents.tripBehaviors.mulitnomialLogit.generate_secondary_activities = "false"
+beam.agentsim.agents.tripBehaviors.mulitnomialLogit.intercept_file_path = ${beam.inputDirectory}"/activity-intercepts.csv"
+beam.agentsim.agents.tripBehaviors.mulitnomialLogit.activity_file_path = ${beam.inputDirectory}"/activity-params.csv"
+beam.agentsim.agents.tripBehaviors.mulitnomialLogit.additional_trip_utility = "double | 0.0"
+beam.agentsim.agents.tripBehaviors.mulitnomialLogit.max_destination_distance_meters = "double | 32000"
```

Comment:
This sets a maximum distance in looking for places to participate in secondary activities. Increasing it increases the maximum and mean trip distance for secondary activities.